### DISCUSSION TITLE
Update for compatibility with Laravel 10

### DISCUSSION THREAD

**Discussion by DmitryGaluschkin** [orchidsoftware/platform#2536] [General]
Hi Team. 
Do you have a plan for update platform for compatibility with Laravel version 10?

**Comment by tabuna** [comment]
Of course, we will support Laravel 10 in the next release

**Reply by WhiskeyM1ke** [reply]
> Of course, we will support Laravel 10 in the next release

Has this update been released yet?

**Reply by arassociates** [reply]
Hey, you could try pointing to master until tag is created for Laravel 10

**Reply by brandonyoungdev** [reply]
When do you think the next release will be ready?
